### DISCUSSION TITLE
2024-03-10 오프라인 회의

### DISCUSSION THREAD

**Discussion by MoonJeWoong** [Dev-On-Play/spring-project#7] [오프라인 회의록]
오프라인 회의

**Comment by MoonJeWoong** [comment]
- 모각코 관리 프로그램

- 로그인(소셜로그인)
    - 다중가입을 방지하기 위해 이메일을 체크한다.
    - 기타 개인정보는 수집하지 않는다.
        - 이메일 주소 + 이름 정도만 수집한다.

- 마이페이지
    - 자신 참석했던 모각코 내역을 볼 수 있다.
    - 앞으로 참석 예정인 모각코 일정을 볼 수 있다.
    - 알림 메뉴를 만들어 로그인하면, 나와 관련된 알림을 볼 수 있다.

- 메인페이지
   - 전체 카테고리를 보여준다
   - 전체 모각코를 보여준다. 
   - (*) 검색창이 있다.

- 생성
    - 모각코 생성시 이름/모임일/모임시간(시작~종료)/참여인원/최소인원/상세내용/분야 등을 설정할 수 있다.
        - 모임일을 언제까지 설정할 수 있는지(생성일 로부터 2주일 까지 설정할 수 있다.)
        - 취소 / 모집중 / 모집 완료 / 모각코 완료 
        - 생성한 사람이 모각코장이 된다.
    - 모각코 분야를 설정할 수 있다.
        - 카테고리 기능 
    - 모임 조건(최소인원 미달성 등)이 모임일까지 달성/달성되지 않을 경우 모임은 취소/성공 된다.
        - 취소/성공 상태로 만든다.
        - 배치는 시간단위로, 마감기준을 시간단위로 한다.
    - (*) 알림 기능  

- (*) 수정
    - 수정

- 참여
    - 참여 또는 취소를 자유롭게 할 수 있다.
    - 미참석시 패널티를 부여 할 수 있다.(자동 부여)
    - 참여와 관련된 질의를 할 수 있는 댓글 기능을 사용할 수 있다.(메신저 대체)
    - 평점이 있다.(당근온도)
    - 참여한 모각코에 파일첨부를 할 수 있다.
        - 모각코에 사용된 자료(크기 및 확장자 제한)
    - 모각코 검색 기능(카테고리) 
        - (*) 타이틀 검색(elk)

- (*) 관리자
    - 모각코장/모각코 참여인원 등, 전체 사용자에 대한 관리를 할 수 있다.
    - 전체 모각코에 대한 일정을 파악할 수 있다.
    - 생성된 모각코에 대해 평점을 관리할 수 있다.

**Comment by hdwhdwhdw** [comment]
erd 클리우드, 피그마, 테일윈드 공용파일 생성하기

**Reply by hdwhdwhdw** [reply]
젠킨스/깃허브 액션

**Reply by hdwhdwhdw** [reply]
마리아 디비 서버 구축
